### PR TITLE
Prototype movecards aa1

### PR DESCRIPTION
**Pull request by Theralun**
**Checklist:**

 
 - [ ] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Documentation (non-breaking change which adds documentation)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently there is no feature for movable elements.

**What is the new behavior (if this is a feature change)?**
- Created a prototype for movable elements.

**Other information**:
-

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,43 +1,42 @@
...
-        "node_modules"
-    ],
-    "cSpell.words": [
-        "browserslist",
-        "homescreen",
-        "precached",
-        "renamer",
-        "webfonts"
-    ]
+  "editor.fontFamily": "Fira Code, Consolas, 'Courier New', monospace",
```
Comment (LucaAllmaier) [inline .vscode/settings.json:2]: please use " " instead of ' '

Comment (Aichnerc) [reply .vscode/settings.json:2]: This does not work, because `'` are within a `"`.

**Review by LucaAllmaier — APPROVED** [review]
LGTM

**Review by Aichnerc — APPROVED** [review]
LGTM

**Pull request merged**